pkg/validation/validate.go

```diff
@@ -0,0 +1,31 @@
+package validation
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+const (
+	qcow2Ext = ".qcow2"
+	tar      = ".tar"
+	gzip     = ".gz"
+	xz       = ".xz"
+)
+
+func IsQcow2(file string) bool {
+	return filepath.Ext(file) == qcow2Ext
+}
+
+func IsTarBall(file string) bool {
+	return strings.Contains(file, tar) && filepath.Ext(file) == gzip || filepath.Ext(file) == xz
```

Comment:
what if the file name is _.targetFile.gz_ # hidden file. Corner case for sure but do we care for now?

Reply:
ah, right :) will fix

Reply:
also if the file name is _foo.tar_ shouldn't that be considered a tarball? If not then the func name should be changed to something like `IsCompressedTarball`